translations/qgc_source_zh_CN.ts

```diff
@@ -7007,7 +7007,7 @@ Click Ok to start the auto-tuning process.
     <message>
       <location filename="../src/FlightDisplay/FlyViewPreFlightChecklistPopup.qml" line="21"/>
       <source>Pre-Flight Checklist</source>
-      <translation type="unfinished">Pre-Flight Checklist</translation>
+        <translation>飞行前检查单</translation>
```

Comment:
Have you done manually on via an translation tool ? the indentation appears wrong